src/metabase/models/user.clj

```diff
@@ -518,6 +518,22 @@
   :type       :boolean
   :default    true)
 
+(defsetting browse-models-sort-column
+  (deferred-tru "User preference for the sort column for the 'Browse models' table")
+  :user-local :only
+  :export?    false
+  :visibility :authenticated
+  :type       :string
+  :default    "collection")
+
+(defsetting browse-models-sort-direction
+  (deferred-tru "User preference for the sort direction for the 'Browse models' table")
+  :user-local :only
+  :export?    false
+  :visibility :authenticated
+  :type       :string
+  :default    "asc")
+
```

Comment:
These are so closely related I thought about storing them together as a JSONified string, or as a string like "+name" or "-name", but this flat, explicit approach seems simplest

Reply:
We often will used a closed set of valid choices here. No worries not to, but need to make sure that the FE can handle invalid input.

Reply:
If the sort column is invalid - neither "asc" or "desc" - then the FE defaults to ascending

Reply:
and also the column to sort on? (no need to respond, just making sure we are defensive)

Reply:
Ah yes, to correct my earlier statement: if the sort _direction_ in the API is invalid (or absent) we default to `asc`.

If the sort _column_ in the API is invalid (or absent), we default to `collection`.